app/assets/stylesheets/application/_colors.scss

```diff
@@ -0,0 +1,2 @@
+$info-color: #3498db;
```

Comment:
It would be provided if we have used boostrap and bootstrap-flatty sass versions

I am going to add the bootstrap variables in another refactor so we can improve the style